rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -326,7 +331,17 @@ def getRelatedSampleIdentity(self, index):
 class SampleIterator:
```

Comment:
In general, do we have any coding guidelines or similar for python?

Reply:
Let's follow the official Python style. So far I've only applied  with regards to naming conventions. https://www.python.org/dev/peps/pep-0008/